src/platform/nxp/zephyr/wifi/ConnectivityManagerImplWiFi.cpp

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
This `wifi` implementation looks almost identical to nRF Connect's implementation. Do you think it could be unified by moving it (or a part of that) to `src/platform/Zephyr/wifi` and building in both platforms? We initially put it in `src/platform/nrfconnect` as we used proprietary Wi-Fi APIs back then but now that we switched to Zephyr's net management APIs, it could probably be unified. That way we may help each other in the future when bugs are found in the shared code.

Reply:
Hi Damian, I agree, this is also the purpose of this PR, to see if we can put even more modules in common for zephyr platforms.
We already made a few fixes, after running internal certification tests, thus the minor differences.

If that makes sense to you, I can do the change in another PR, I'd prefer to not stack changes in this PR as it's already kinda heavy to review.

Reply:
Yes, works for me. You can probably file an issue for that so it's not forgotten.

Reply:
Done: https://github.com/project-chip/connectedhomeip/issues/31651

Reply:
Thanks @axelnxp, looking forward for PR. What about the features defined in chip-module/KConfig or KConfig.features? Originally these KConfig options were created for nRF Connect platform, now they are copied to another file almost without modifications. I wonder if we could move them to common generic Zephyr port as well and simplify maintenance in the future?

Reply:
chip-module/Kconfig is already unified (config/zephyr/Kconfig). Only platform-specific Kconfig are separated.